sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -930,6 +930,45 @@ object RapidsConf {
       |  --conf spark.executorEnv.LIBCUDF_KERNEL_CACHE_PATH="/tmp/cudf-$USER"
       |  ```
       |""".stripMargin)
+
+      printSectionHeader("RAPIDS-Plugin Cache Serializer")
+      // scalastyle:off line.size.limit
+      println("""
+      |  Apache Spark provides an important feature to cache intermediate data and provide
```

Comment:
I really hate having so much non-auto generated info in here.  Even the JIT kernel cache path feels a bit like it is pushing things.

What is the plan for documenting things like this in the future?
1. Caching
2. Alluxio
3. ShuffleManager (not really documented anywhere except in on-prem getting started)
4. UDFs which have their own file, but there is really no documentation for the UDF compiler

We really should be consistent, and I understand if some features are going to be more important to users than others, but if a user has to look in 5 different places to see what we offer I think it is much worse then a long side panel.  Especially if we offer huge amount of functionality that has to be enabled/disabled separately.

If we want it all here then I would like to see us find some way to template the auto generated docs instead of putting it all in a file that is already close to a thousand lines.  I personally would prefer to split it up and perhaps have some sub-directories for distinctly different features. Possibly even with SQL being one of them, because RapidsConf really should probably be more like RapidsSQLConf.

Reply:
@sameerz can we have a discussion on this? @revans2 brings up good points about the configs.md being really long. We could think about splitting it up into multiple files which could go under sub-menus on the left panel?